src/lumigo_tracer/lambda_tracer/lambda_reporter.py

```diff
@@ -168,36 +182,135 @@ def report_json(
     return duration
 
 
+def get_span_priority(span: Dict[Any, Any]) -> int:
+    if span.get("type") == FUNCTION_TYPE:
+        return 0
+    if is_span_has_error(span):
+        return 1
```

Comment:
It will be hard to explain this to the user.
I'm suggesting a second ordering - by time!
first include error spans, but if there are too much - include the chronological first of these..

Reply:
Im not sure I understand, you want to add more filed to sort by no?
how its easier to explain?

Reply:
exmple: 
assume an invocation with 1000 successful HTTP requests.
I think that it makes sense that Lumigo will present the first 200, and not random 200 (as currently happens)

Reply:
I understand that, but like I said your are suggestion something more to sort by so by definition its something more we need to explain => meaning it will be harder, no?

we can do it but I don't think its so important.